Anytype/Sources/PresentationLayer/Flows/HomeFlow/HomeCoordinatorViewModel.swift

```diff
@@ -233,14 +234,25 @@ final class HomeCoordinatorViewModel: ObservableObject,
     
     func onSearchSelected() {
         AnytypeAnalytics.instance().logScreenSearch()
-        showSearchData = ObjectSearchModuleData(
-            spaceId: activeWorkspaceStorage.workspaceInfo.accountSpaceId,
-            title: nil,
-            onSelect: { [weak self] data in
-                AnytypeAnalytics.instance().logSearchResult()
-                self?.openObject(screenData: data.editorScreenData)
-            }
-        )
+        
+        if FeatureFlags.newGlobalSearch {
+            showGlobalSearchData = GlobalSearchModuleData(
+                spaceId: activeWorkspaceStorage.workspaceInfo.accountSpaceId,
+                onSelect: { [weak self] screenData in
+                    AnytypeAnalytics.instance().logSearchResult()
```

Comment:
I think you can move analytics to module (logScreenSearch and logSearchResult)

Reply:
Moved